[docs] Explain that you can't update the primary key columns in YCQL UPDATE (#17188)
yugabyte/yugabyte-db: docs/content/preview/api/ycql/dml_update.md

  them. Using a mix of statements where some have `USING TIMESTAMP` and others do not will lead to
  very confusing results.
- Updating rows `USING TTL` is not supported on tables with [transactions enabled](./../ddl_create_table#table-properties-1).

> - You cannot update the columns in the primary key. You have to delete the row and insert a new row in this case.

Reviewer: ```suggestion
- You cannot update the columns in the primary key. As a workaround, you have to delete the row and insert a new row.
```

> - You cannot update the columns in the primary key. As a workaround, you have to delete the row and insert a new row.

Author: ```suggestion
- You cannot update the columns in the primary key. As a workaround, you have to replace the whole row.
```
